_posts/2023-01-23-there-is-no-cloud.md

```diff
@@ -0,0 +1,294 @@
+# There Is No Cloud, It's Just Someone Else's Computer
+
+### By Sergei Vorobyov, Solita/Stockholm
```

Comment:
No need to put author info as a header, the default blog rendering will already put author info at the top

Reply:
Line deleted